packages/rum/package.json

```diff
@@ -25,11 +25,11 @@
     "serve": "npm run script serveE2e ./ 8000",
```

Comment:
This can be removed as well. It doesn't work with the current refactoring since `serveE2e` is removed.

Reply:
good catch 👍